buildstockbatch/workflow_generator/residential_hpxml.py

```diff
@@ -317,6 +318,7 @@ def create_osw(self, sim_id, building_id, upgrade_idx):
             'include_timeseries_airflows': False,
             'include_timeseries_weather': False,
             'timeseries_timestamp_convention': 'end',
+            'timeseries_num_decimal_places': 3,
```

Comment:
Remove this, otherwise we break backward compatibility. On the other hand if this was a required argument, then we may _need_ to break backward compatibility (i.e., old versions of ResStock _won't_ work with newer version of the workflow generator).